api/src/main/java/org/motechproject/nms/api/web/OpsController.java

```diff
@@ -80,4 +92,65 @@ public void startUpkeep() {
         LOGGER.info("/upkeep");
         eventRelay.sendEventMessage(new MotechEvent(KilkariConstants.SUBSCRIPTION_UPKEEP_SUBJECT));
     }
+
+    @RequestMapping(value = "/createUpdateFlw",
```

Comment:
@VishnuVimal @atish160384 please note this change in endpoint. should be minimal change (from /addFlw) to update in your client if you are using the latest build from here.